src/components/ErrorState/index.tsx

```diff
@@ -53,6 +56,8 @@ export const ErrorState = ({
   buttonLabel,
   disabled,
   block,
+  buttonOnClick,
```

Comment:
Não é melhor algo como `onCtaClick` ou `onButtonClick`? Fica mais declarativo.

Reply:
I fix this in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/39/commits/ffacbff9d522e5bdbf5c7482b299983a0487a60a)